ksa_compliance/ksa_compliance/doctype/zatca_business_settings/zatca_business_settings.py

```diff
@@ -286,7 +288,7 @@ def on_trash(self) -> None:
 
     def create_tax_account(self) -> str:
         account_doc = cast(Account, frappe.new_doc("Account"))
-        parent_account = frappe.get_value("Account", {"company": self.company, "account_name": "Duties and Taxes"}
+        parent_account = frappe.get_value("Account", {"company": self.company, "account_name": _("Duties and Taxes")}
```

Comment:
Please add a comment explaining the importance of the translation with `_`. Otherwise the reader might assume this is done by mistake and is a bug.